src/windows/pdb.rs

```diff
@@ -599,9 +757,13 @@ mod tests {
         }
     }
 
-    fn test_file(name: &str) {
-        let dll = name.to_string() + ".dll";
-        let out = get_new_bp(&dll);
+    fn test_file(name: &str, check_multiplicity: bool) {
```

Comment:
Boolean arguments make code harder to read, better use an enum representing the test flags and have a CheckMultiplicity flag that you can use for this.